recipes-asteroid/asteroid-launcher/asteroid-launcher_git.bb

```diff
@@ -23,4 +29,14 @@ do_install_append() {
     install -d ${D}/usr/lib/systemd/user/default.target.wants/
     cp ../asteroid-launcher.service ${D}/usr/lib/systemd/user/
     ln -s ../asteroid-launcher.service ${D}/usr/lib/systemd/user/default.target.wants/asteroid-launcher.service
+
+    install -d ${D}/etc/systemd/system/user@.service.d/
+    cp ../localeEnv.conf ${D}/etc/systemd/system/user@.service.d/locale.conf
```

Comment:
This file should be installed in the new supported languages recipe. It also has nothing to do here